runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -328,6 +325,56 @@ public void onChatMessage(ChatMessage event)
 		}
 	}
 
+	@Subscribe
+	public void onGameObjectSpawned(GameObjectSpawned event)
+	{
+		final GameObject gameObject = event.getGameObject();
+		final WorldPoint bombLocation = gameObject.getWorldLocation();
+
+		switch (gameObject.getId())
+		{
+			case ObjectID.CRYSTAL_BOMB:
+				bombs.put(bombLocation, new CrystalBomb(gameObject));
+				break;
+		}
+
+	}
+
+	@Subscribe
+	public void onGameTick(GameTick event)
```

Comment:
This entire method becomes unnecessary if you use `client.getTickCount()` to measure ticks the bombs are alive for.